src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -110,6 +110,15 @@ const LaunchAnalysisModal = ({
           p(['Note that metadata about this run will be stored in the US.'])
         ])]) : 'Loading...'
     ]),
+    div(['Compute for this workflow will occur ', span({ style: { fontWeight: 'bold' } }, 'by default '), 'in the region:']),
+    div({ style: { margin: '1rem 0 1.5rem' } }, [
+      location ? h(Fragment, [span({ style: { marginRight: '0.5rem' } }, [flag]),
+        h(Fragment, [span({ style: { marginRight: '0.5rem' } }, [computeRegion])])]) : 'Loading...'
+    ]),
```

Comment:
I had a little difficult time parsing the nesting above. I wonder if the following may be a bit easier:

```suggestion
    div({ style: { margin: '1rem 0 1.5rem' } }, [
      location ?
        h(Fragment, [
          span({ style: { marginRight: '0.5rem' } }, [flag]),
          h(Fragment, [span({ style: { marginRight: '0.5rem' } }, [computeRegion])])
        ]) :
        'Loading...'
    ]),
```

Reply:
Done.